### PR TITLE
renamer: fix persist and pre_install sections

### LINKED ISSUE #9312: Renamer‘s persets is not in the persist.
## Bug Report

**Package Name:** [renamer]

### Current Behaviour

Renamer‘s persets is not in the persist.

### Expected Behaviour

Add the Renamer‘s persets to persist.

### System details

**Windows version:** 10

**OS architecture:** 64bit
**PowerShell version:** 
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1682

#### Scoop Configuration
```json
{
    "lastUpdate":  "2022-09-24T22:06:57.0886998+08:00",
    "SCOOP_REPO":  "https://ssh.fastgit.org/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master",
    "aria2-warning-enabled":  false,
    "aria2-enabled":  false
}
```

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #9312

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/renamer.json [hunk 1]
```diff
@@ -5,15 +5,21 @@
     "license": "CC-BY-NC-ND-3.0|Shareware",
     "url": "https://www.den4b.com/download/renamer/portable/7.4?token=scoop#/dl.zip",
     "hash": "1a47a345773ef44222efbe1c1f4c58312470ebd5965fc60d0555d932d620acc2",
-    "pre_install": "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item \"$dir\\Settings.ini\" | Out-Null }",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item \"$dir\\Settings.ini\" -ItemType File | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\Presets\")) { New-Item \"$dir\\Presets\" -ItemType Directory | Out-Null }"
```
Comment (chawyehsu) [inline bucket/renamer.json:10]: You don't have to pre-create the directory to persist.

Comment (Zliced13) [reply bucket/renamer.json:10]: Oh your right, I'll fix that.

**Comment by Zliced13** [comment]
Done

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**